indra/databases/cbio_client.py

```diff
@@ -265,6 +356,34 @@ def get_ccle_lines_for_mutation(gene, amino_acid_change):
     cell_lines = df['case_id'].unique().tolist()
     return cell_lines
 
+
+def get_ccle_cna(gene_list):
+    """Return a dict of CNAs in given genes and cell lines from CCLE.
+
+    This is a specialized call to get_mutations tailored to CCLE cell lines.
+
+    Parameters
+    ----------
+    gene_list : list[str]
+        A list of HGNC gene symbols to get mutations in
+    cell_lines : list[str]
+        A list of CCLE cell line names to get mutations for.
+    mutation_type : Optional[str]
+        The type of mutation to filter to.
+        mutation_type can be one of: missense, nonsense, frame_shift_ins,
+                                     frame_shift_del, splice_site
+
+    Returns
+    -------
+    profile_data : dict[dict[int]]
+        A dict keyed to cases containing a dict keyed to genes
+        containing int
+    """
+    profile_data = get_profile_data(ccle_study, ['BRAF', 'PTEN'],
+                                    'COPY_NUMBER_ALTERATION', 'all')
```

Comment:
- Looks like the parameters are hard coded here instead of the actual input arguments
- I think this function should take a cell_lines argument so that not every cell line in CCLE is returned every time, but only the ones specified in cell_lines
- The documentation Parameters should match the actual input arguments

Reply:
Thanks for catching this!
- Fixed hardcoded list in a610f4c
- Updated tests in  5860b83
- parameters matched to documentation